resources/views/bread/browse.blade.php

```diff
@@ -39,9 +41,9 @@
 
                                                     @foreach($data->{$row->field} as $item)
                                                         @if($item->{$row->field . '_page_slug'})
-                                                        <a href="{{ $item->{$row->field . '_page_slug'} }}">{{ $item->{$row->field}  }}</a>@if(!$loop->last), @endif
+                                                        <a href="{{ $item->{$row->field . '_page_slug'} }}">{!! $item->{$row->field} !!}</a>@if(!$loop->last), @endif
```

Comment:
https://github.com/the-control-group/voyager/pull/695#issuecomment-277903990
https://github.com/the-control-group/voyager/pull/695#issuecomment-277912731